training/README.md

```diff
@@ -0,0 +1,60 @@
+Training in Digital Publishing
+===========================
+
+:warning: **This section is very much a work in progress. Links may not resolve, and content linked to may be unfinished** 
+
+When joining Digital Publishing, there can be a lot to learn! We want to give you the benefit of all our previous learning, so we've tried to gather it in one place.
+
+We have collected resources that give you an idea of [all the modules](#modules) some of which you'll be undertaking over your first few weeks. These may also contain links to resources the team have found useful over the years to get more insight on the topics.
+
+If you come across a resource that really helped you understand some technology we use, or how we work, please do add a link to it in the relevant document or start a new doc if the topic is not yet covered.
+
+Modules
+----------------------------
+#### Team culture & process
+* [Culture and Inclusion]()
+* [What is Digital Publishing?](culture-and-process/DIGITAL_PUBLISHING.md) 
+* [Developer Processes]()
+
+#### Architectural principles & approach
+* [12 Factor App principles]()
+* [Security]()
+* [Data modeling, what databases we use and when]()
+
+#### Our services
+* [Core Services]()
+* [Microservices & Customise my Data]()
+* [Cloud technologies and Infrastructure (should this move to architecture?)](services/INFRASTRUCTURE.md)
+* [Platform Services]()
+
+#### Technologies we use
+* [How we choose technologies]()
+* Go
+    * [Intro to Go]()
+    * [Concurrency]()
+* AWS
+    * [Intro to AWS]()
+
+#### Testing
+* [Our testing approach]()
+* [Unit testing in Go]()
+* [Unit testing in Java]()
```

Comment:
We could add a section on Frontend unit testing in React (Jest), or something like that